### PR TITLE
#1497 The feature to compress pictures

### LINKED ISSUE #1497: Merge and compress pictures
**Actual behaviour**
There are no particular options to compress or merge pictures. 

**Expected behaviour**
- [ ] Whenever user selects more than 1 pic there can be a feature of Merge and edit. Also, there must be different ways in which one can merge or it can be freehand 

- [x] Feature to Compress pictures. Whenever user selects a pic in the options menu, there can be Compress option. It will allow user to compress the picture to any no. of pixels and size

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by iamjaspreetsingh**
Fixed #1497, one part is completed.

Changes: Created a new activity and few dialog boxes for the feature to compress images to smaller no. of pixels or size (both the options are given separately). The icon of Compress is given below each picture alongside Edit icon.

Screenshots of the change: 
34470314-fab06d6e-ef54-11e7-8be2-6e03e49923c1
34470315-fb08b0b4-ef54-11e7-9a75-1dfea3dcc2e6
34470316-fb470206-ef54-11e7-9bc1-14439857785b
34470317-fb848c3e-ef54-11e7-86ea-bfdb8bf681d3

Kindly review it please. 
Thank you

### CODE REVIEW

**Comment by iamjaspreetsingh** [comment]
There is no conflict as such. 
Please check..

**Comment by abishekvashok** [comment]
There are @iamjaspreetsingh, can you update your fork and make the changes please?
`git pull upstream --rebase` and do a force push where upstream is this repo.

**Comment by iamjaspreetsingh** [comment]
Thanks @abishekvashok I am trying to resolve it soon

**Comment by iamjaspreetsingh** [comment]
Conflicts are now resolved.
Please check and kindly review.

File: app/src/main/java/org/fossasia/phimpme/editor/CompressImageActivity.java [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+import java.util.ArrayList;
+
+import id.zelory.compressor.Compressor;
+
+public class CompressImageActivity extends AppCompatActivity {
+    public String saveFilePath;
+    public static final String EXTRA_OUTPUT = "extra_output";
+    public int percentagecompress=0;
+
+
```
Comment (mayank8318) [inline app/src/main/java/org/fossasia/phimpme/editor/CompressImageActivity.java]: Avoid extra lines here

File: app/src/main/java/org/fossasia/phimpme/editor/CompressImageActivity.java [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+        initView();
+        
+        Button size=(Button)findViewById(R.id.size);
+        Button dimension=(Button)findViewById(R.id.bypixel);
+        ImageButton cancel=(ImageButton)findViewById(R.id.edit_cancel);
+
+        size.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                compressbysize();
```
Comment (mayank8318) [inline app/src/main/java/org/fossasia/phimpme/editor/CompressImageActivity.java]: Camel case

File: app/src/main/java/org/fossasia/phimpme/editor/CompressImageActivity.java [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+            }
+        });
+        dimension.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                compressbydim();
```
Comment (mayank8318) [inline app/src/main/java/org/fossasia/phimpme/editor/CompressImageActivity.java]: Camel case

**Comment by sauravvishal8797** [comment]
@iamjaspreetsingh Please update the branch with latest changes.

**Comment by iamjaspreetsingh** [comment]
@sauravvishal8797 updated the branch.

**Comment by iamjaspreetsingh** [comment]
@sauravvishal8797 kindly check it now:)

**Pull request merged**